lib/OpenQA/WebSockets/Client.pm

```diff
@@ -21,37 +21,33 @@ use Carp 'croak';
 use OpenQA::Client;
 use OpenQA::Utils 'service_port';
 
-has host   => sub { $ENV{OPENQA_WEB_SOCKETS_HOST} };
-has client => sub { OpenQA::Client->new(api => shift->host // 'localhost') };
-has port   => sub { service_port('websocket') };
+has host   => sub ($self) { $ENV{OPENQA_WEB_SOCKETS_HOST} };
+has client => sub ($self) { OpenQA::Client->new(api => $self->host // 'localhost') };
+has port   => sub ($self) { service_port('websocket') };
 
 my $IS_WS_SERVER_ITSELF;
-sub mark_current_process_as_websocket_server { $IS_WS_SERVER_ITSELF = 1; }
-sub is_current_process_the_websocket_server  { return $IS_WS_SERVER_ITSELF; }
+sub mark_current_process_as_websocket_server() { $IS_WS_SERVER_ITSELF = 1; }
+sub is_current_process_the_websocket_server()  { return $IS_WS_SERVER_ITSELF; }
```

Comment:
Odd that perltidy won't insert a whitespace here between the name and the signature.

Reply:
I added an extra style check which ensures this, same as done for os-autoinst. Also I adjusted the style of existing and these occurences accordingly.